app/controllers/secure_controller.js

```diff
@@ -20,11 +20,14 @@ const paths = require('../paths')
 exports.new = async function (req, res) {
   const chargeTokenId = req.params.chargeTokenId || req.body.chargeTokenId
   const correlationId = req.headers[CORRELATION_HEADER] || ''
+  var chargeId
+  var gatewayAccountId
+  var gatewayAccountType
```

Comment:
[minor] `var` -> `let`

Reply:
what's the advantage of `let` here?

Reply:
In this particular case it doesn't make functional difference.
`let` is always more appropriate to use in terms of scoping. (`var` should really be avoided).
It's a minor comment - it doesn't block this PR. I was concerned about the other thing (that's why I haven't approved previously).